src/commands.ts

```diff
@@ -296,22 +296,29 @@ export function findText(args: FindTextArgs) {
 		pos = pos || 0;
 
 		// Search forward from position in string
-		let indexOf = str.substring(pos).search(RegExp(regex));
+		let index = str.substring(pos).search(regex);
```

Comment:
I see you remove `RegExp(regex)` here, which I think will result in searching the raw string instead of a regex. Is it removed by mistake?

Reply:
Good eye.

- I was digging into the documentation and saw that [the argument to `search` is implicitly converted to a `RegExp`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/search#parameters).
- [The same is true for `matchAll`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/matchAll#parameters). The problem for `matchAll` is that this is apparently a recent change and is not yet reflected in this version of TypeScript, so I had to perform the conversion manually.

I think for consistency sake, I will push a commit in which both calls include an explicit call to `RegExp`.

Reply:
Oh you are right. Using search directly should work as well